docker-files/zeeguu-api-core/Dockerfile

```diff
@@ -24,17 +25,39 @@ RUN python setup.py install
 # https://github.com/zeeguu-ecosystem/Python-Wordstats/blob/master/wordstats/config.py#L1
 # under /opt/Zeeguu-API/wordinfo.db
 COPY --chown=www-data:www-data Zeeguu-API /opt/Zeeguu-API
+RUN rm -rf /opt/Zeeguu-API/.git
 WORKDIR /opt/Zeeguu-API
 RUN pip install -r requirements.txt
-RUN python setup.py install
+
+#ML: This is needed because the translators installer installs nltk in ~ (which in the
```

Comment:
I think this would look better as ``# NOTE(mircealungu): ...`` since it will be helpful for others in the future if they have questions to ask.